src/components/chart/column-chart.js

```diff
@@ -15,31 +15,33 @@ class ColumnChart {
         };
     };
 
-    // Set the point padding between each bar to be 3% (an overall gap of 6%)
-    // For charts with fewer than 5 categories, we use a wider point padding of 4% (8% gap between bars)
-    // For cluster charts we use 0 for the point padding and a group padding of 4% (8% gap between bars)
-    updatePointPadding = (config, currentChart, stackedLayout, numberOfExtraLines) => {
+    // Set the point padding between each bar
+    // For charts with fewer than 5 categories, we use a wider point padding of 2%
+    // For cluster charts we use 0 for the point padding and a group padding of 2%
```

Comment:
This comment now isn't that clear, what is the default now? is 2% wider than it?